release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[[ocp-4-9-33]]
+=== RHSA-2022:2205 - {product-title} 4.9.33 bug fix and security update
```

Comment:
This is always the images advisory, so it should be 

```suggestion
=== RHBA-2022:2206 - {product-title} 4.9.33 bug fix and security update
```

Reply:
Ack'ed